CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@
 
 Crayons is a component library based on Web Components. The goal of Crayons is to provide the developer with easy to use web components by reducing the number of lines of code while giving your app a Freshworks product like user interface.
 
-Crayons are built using StencilJS which provides all features of web component.
+Crayons is built using StencilJS which provides all the features of a web component.
```

Comment:
Could we modify this content to one of the following? 

- Crayons is a component library based on Web Components. The goal of Crayons is to provide developers with easy to use web components that give apps a Freshworks product like user interface, while reducing the number of lines of code. Crayons is developed using StencilJS that helps build standard-based Web Components.

- Crayons is a component library based on Web Components. The goal of Crayons is to provide developers with easy to use web components that reduce the number of lines of code involved in giving apps a Freshworks product like user interface. Crayons is developed using StencilJS that helps build standard-based Web Components.

Reply:
I wouldn't mind Zach.